src/SWP/Component/Bridge/Model/AuthorsAwareTrait.php

```diff
@@ -50,4 +50,11 @@ public function hasAuthor(AuthorInterface $author): bool
     {
         return $this->authors->contains($author);
     }
+
+    public function getAuthorsNames()
```

Comment:
this method signature should be declared in the `AuthorsAwareInterface` interface, also return type is missing in the method signature itself